vpr/src/pack/cluster.cpp

```diff
@@ -2092,25 +2124,15 @@ static void update_cluster_stats(const t_pack_molecule* molecule,
                                  const bool timing_driven,
                                  const bool connection_driven,
                                  const int high_fanout_net_threshold,
-                                 const SetupTimingInfo& timing_info) {
-    /* Updates cluster stats such as gain, used pins, and clock structures.  */
+                                 const SetupTimingInfo& timing_info,
+                                 AttractionInfo& attraction_groups) {
+    /* Updates cluster stats such as gain, used pins, and clock structures. Also
```

Comment:
Write better overall comment - a molecule has been added to the cluster, this updates stats on that cluster such as used pins, etc. and makes calls to make sure gains of other atoms are adjusted to reflect the new cluster content